docs/mobile_data_collected.md

```diff
@@ -0,0 +1,173 @@
+#Mobile Data Collected
+
+The Datadog Real User Monitoring SDK generates six types of events:
+
+| Event Type | Description                                                                                                                                                                                                                                                   |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Session    | Session represents a real user journey on your mobile application. It begins when the user launches the application and it is kept live as long as the user stays active. During the user journey, all RUM events generated as part of the session will share the same `session.id` attribute.
+ |
+| View       | View represents a unique screen (or screen segment) on your mobile application. Individual ViewControllers are classified as distinct views. While a user stays on a view, RUM event attributes (Errors, Resources, Actions) get attached to the View with a unique `view.id`                                   |
+| Resource   | Resource represents network requests to first-party hosts, APIs, 3rd party providers and libraries in your mobile application. All requests generated during a user session are attached to the view with a unique `resource.id`                                                      |
+| Error     | Error represents an exception or crash (fatal error) emitted by the mobile application attached to the view it is generated in.                                                                                                                                            |
```

Comment:
`fatal error` may be misunderstood due to `fatalError()` Swift function causing a particular type of crash. I'd remove it at all and not specify what is a "crash" (every iOS dev knows that).
```suggestion
| Error     | Error represents an exception or crash emitted by the mobile application attached to the view it is generated in.                                                                                                                                            |
```